### PR TITLE
Create main page layout

### PR DESCRIPTION
**Pull request by ioannanedelcu**

### CODE REVIEW

File: project/pom.xml [hunk 1]
```diff
@@ -4,7 +4,7 @@
   <modelVersion>4.0.0</modelVersion>
 
   <groupId>com.google.sps</groupId>
-  <artifactId>portfolio</artifactId>
+  <artifactId>project</artifactId>
```
Comment (conradgrobler) [inline project/pom.xml:7]: Perhaps use a more descriptive name.

File: project/target/maven-archiver/pom.properties [hunk 2]
```diff
@@ -0,0 +1,5 @@
+#Generated by Maven
```
Comment (conradgrobler) [inline project/target/maven-archiver/pom.properties:1]: It seems that you might be missing a `.gitignore' file somewhere, as I would expect this generated code under target not to be included.

File: project/target/project-1/profile.html [hunk 3]
```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
```
Comment (conradgrobler) [inline project/target/project-1/profile.html:1]: I would not expect this to be under the target directory, as it is generated. This should probably be in the same location as the index.html (project/src/main/webapp) together with its associated images, script and css.

File: project/src/main/webapp/index.html [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+
+  </head>
+
+  <body>
+
+    <!-- Header -->
+    <div class="header">
+      <div style="border-style:outset;">
+        <h1>TheGlobetrotter</h1>
+        <p>Plan your trip and share it with your firends!</p>
```
[the remark this answers is not in the document]
Comment (ioannanedelcu) [reply project/src/main/webapp/index.html:20]: Done

File: project/src/main/webapp/index.html [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+        <p>Some text ....</p>
+        <p>Some text ....</p>
+        <p>Some text ....</p>
+        <p>Some text ....</p>
+      </div>
+    </div>
+
+  </body>
+
+</html>
```
[the remark this answers is not in the document]
Comment (ioannanedelcu) [reply project/src/main/webapp/index.html:58]: Done

File: project/pom.xml (hunk 1 above)
Comment (ioannanedelcu) [reply project/pom.xml:7]: Done

File: project/target/maven-archiver/pom.properties (hunk 2 above)
Comment (ioannanedelcu) [reply project/target/maven-archiver/pom.properties:1]: Done

**Pull request merged**